src/Components/Form/Form.js

```diff
@@ -4,10 +4,6 @@ import transportOptions from '../../assets/travelModeData';
 import TripETA from '../TripETA/TripETA';
 import Autocomplete from 'react-google-autocomplete';
 
-// import {SearchLocationInput} from '../SearchLocationInput/SearchLocationInput.js'
-// import {SearchLocationInput2} from '../SearchLocationInput2/SearchLocationInput2.js'
```

Comment:
nice work cleaning this up and removing unneeded stuff. My fault for pushing it up with that!